lib/google/longrunning/operations_client.rb

```diff
@@ -347,6 +318,24 @@ def delete_operation \
         @delete_operation.call(req, options)
         nil
       end
+
+      protected
+
+      def default_settings(metadata, lib_name, lib_version)
+        package_version = Gem.loaded_specs['google-gax'].version.version
+
+        google_api_client = ["gl-ruby/#{RUBY_VERSION}"]
+        google_api_client << "#{lib_name}/#{lib_version}" if lib_name
+        google_api_client << "gapic/#{package_version}"
+        google_api_client << "gax/#{Google::Gax::VERSION}"
+        google_api_client << "grpc/#{GRPC::VERSION}"
+        google_api_client.join " "
+
+        headers = { 'x-goog-api-client' => google_api_client }
+        headers.merge! metadata unless metadata.nil?
+
+        Google::Gax.const_get(:CallSettings).new metadata: headers
```

Comment:
So we no longer need to set `timeout`, `page_descriptor`, or `errors` as was previously done in `construct_settings`?

Reply:
Correct. These values will either be passed in differently, or will be removed from 2.0.